processor/src/services/operation.service.ts

```diff
@@ -65,7 +66,7 @@ export class DefaultOperationService implements OperationService {
       },
     });
 
-    const res = await this.processPaymentModification(transactionType, ctPayment.interfaceId as string, requestAmount);
+    const res = await this.processPaymentModification(payment, transactionType, ctPayment.interfaceId as string, requestAmount);
```

Comment:
we are sending the payment object itself and the `ctPayment.interfaceId` which is redundant as if we send the whole payment we don't need to send the interfaceId as it can be obtained from the payment